app/reducers/treatment.js

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios'
```

Comment:
Also, this highly conflicts with the way I am implementing `Counter`. Which is ok, but we may need to resolve conflicts together later today/tomorrow 😃